ui/chat/src/discussion.ts

```diff
@@ -84,11 +84,19 @@ function renderInput(ctrl: Ctrl): VNode | undefined {
 let mouchListener: EventListener;
 
 const setupHooks = (ctrl: Ctrl, chatEl: HTMLElement) => {
+  let tempChat = window.lichess.tempStorage.get("tempChatStorage");
+  if(tempChat !== null && tempChat !== ""){
+    (chatEl as HTMLInputElement).value = window.lichess.tempStorage.get("tempChatStorage")!;
+    window.lichess.tempStorage.set("tempChatStorage", "");
```

Comment:
why set it to empty? I guess why not? Not sure if it's right or wrong, just curious to hear the reasoning

Reply:
You can't .set("a", null) it gives a syntax error. I changed it to .remove() since it makes it much cleaner.

Clearing sessionStorage is a good catch, I added that to the send if.